spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/authService/ISCAuth.java

```diff
@@ -0,0 +1,25 @@
+package com.boundlessgeo.spatialconnect.services.authService;
```

Comment:
Can you put the license at the top and then the package after it?  I believe that's been our convention with the other files